src/boundary/Multigrid.h

```diff
@@ -145,12 +135,12 @@ class Multigrid {
     size_t getLen_bSliceY_joined();
     size_t getLen_bSliceX_joined();
 
-    size_t getFirstIndex_bSliceZ( size_t level);
-    size_t getFirstIndex_bSliceX( size_t level);
-    size_t getFirstIndex_bSliceY( size_t level);
-    size_t getLastIndex_bSliceZ( size_t level);
-    size_t getLastIndex_bSliceX( size_t level);
-    size_t getLastIndex_bSliceY( size_t level);
+    size_t getFirstIndex_bSliceZ(size_t level);
+    size_t getFirstIndex_bSliceX(size_t level);
+    size_t getFirstIndex_bSliceY(size_t level);
+    size_t getLastIndex_bSliceZ(size_t level);
+    size_t getLastIndex_bSliceX(size_t level);
+    size_t getLastIndex_bSliceY(size_t level);
 
     size_t getLen_iList_joined();
     size_t getLen_bList_joined();
```

Comment:
can be marked as const

Reply:
c984e0880c80fa8fa1efe7d96bd03afc2407374c